### PR TITLE
dispatch selector

### PR DESCRIPTION
**Pull request by afinch7**

### CODE REVIEW

File: cli/ops.rs [hunk 1]
```diff
@@ -142,6 +142,13 @@ pub fn dispatch_all(
   (base.sync(), boxed_op)
 }
 
+pub fn op_selector_compiler(inner_type: msg::Any) -> Option<OpCreator> {
+  match inner_type {
+    msg::Any::FetchModuleMetaData => Some(op_fetch_module_meta_data),
+    _ => op_selector_std(inner_type),
```
Comment (ry) [inline cli/ops.rs]: Compiler only does FetchModuleMetaData, right? I think you can do `unreachable!()` here.

Comment (afinch7) [reply cli/ops.rs]: We also need WorkerGetMessage and WorkerPostMessage

Comment (ry) [reply cli/ops.rs]: Oh right - so add those but then branch to unreachable

Comment (afinch7) [reply cli/ops.rs]: also start

**Pull request merged**